launch/launch/main.py

```diff
@@ -31,10 +31,10 @@ def main(argv=sys.argv[1:]):
     parser = argparse.ArgumentParser(
         description='Launch the processes specified in a launch file.')
     parser.add_argument(
-        'launch_file',
+        'launch_files',
```

Comment:
Changing this name also changes the help text (the metavar place holder). The help text should continue to refer to each argument as a launch_file (singular).